### PR TITLE
Add contact us modal

### PR DESCRIPTION
**Pull request by adamduren**

### CODE REVIEW

File: src/contact-us-modal.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<link rel="import" href="../bower_components/polymer/polymer.html">
```
Comment (adamduren) [inline src/contact-us-modal.html]: This is unmodified from the old version

File: src/contact-us-modal.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          <h2>Contact Us</h2>
+          <form is="iron-form" id="form" method="post"
+            action="https://hsj79szpdg.execute-api.us-east-1.amazonaws.com/prod/submit-contact"
+            on-iron-form-response="responseHandler"
+            on-iron-form-error="errorHandler"
+            content-type="application/json">
+
+            <!-- Enter is not triggering submission. Possibly https://github.com/Polymer/core-a11y-keys/issues/3 -->
+            <paper-input name="name" label="Name" required auto-validate></paper-input>
+            <gold-email-input name="email" label="Email" required onblur="this.validate()"></gold-email-input>
```
Comment (zacharytamas) [inline src/contact-us-modal.html]: I think I'd prefer to use Polymer's event handling: `on-blur="_validateForm"` then make the `_validateForm` method on this element. The HTML is supposed to be declarative only.

File: src/contact-us-modal.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      is: 'nl-contact-us',
+
+      properties: {
+        isSubmitted: {
+          type: Boolean,
+          value: false,
+        },
+      },
+
+      click: function() {
```
Comment (zacharytamas) [inline src/contact-us-modal.html]: I'd probably call this `open` since that's what it does.

File: src/contact-us-modal.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        var self = this;
+
+        this.isSubmitted = true;
+        setTimeout(function() {
+          self.isSubmitted = false;
+          self.$.dialog.close();
+        }, 2500);
+      },
+
+      submit: function() {
```
Comment (zacharytamas) [inline src/contact-us-modal.html]: Polymer convention is that these "private" callbacks start with an underscore.

File: src/contact-us-modal.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+      click: function() {
+        this.$.dialog.open();
+      },
+
+      close: function() {
+        var self = this;
+
+        this.isSubmitted = true;
+        setTimeout(function() {
+          self.isSubmitted = false;
```
Comment (zacharytamas) [inline src/contact-us-modal.html]: Does this cause a flash of the form resetting itself before the modal closes?

File: src/index.html [hunk 6]
```diff
@@ -45,11 +45,7 @@
     </div>
 
     <div class="main-cta">
-      <!--<button class="btn btn-outlined">
-        Our Services
-      </button>-->
-
-      <button class="btn">
+      <button class="btn" onclick="contactUs.click()">
```
Comment (zacharytamas) [inline src/index.html]: These `button`s need `cursor: pointer`s on them so they actually look clickable.

**Comment by adamduren** [comment]
Also GH reviews are strange.

**Comment by adamduren** [comment]
Landing now but will address the other comments shortly.

File: src/contact-us-modal.html (hunk 5 above)
Comment (adamduren) [inline src/contact-us-modal.html]: Doesn't appear to

File: src/contact-us-modal.html (hunk 3 above)
Comment (adamduren) [inline src/contact-us-modal.html]: Done

**Pull request merged**